src/runtime/messages/event_aggregator_settings.ts

```diff
@@ -14,6 +14,12 @@ const messageParams = {
 
 const messagePaths = {
   // Generic messages
+  callActivityReached: 'call_activity_reached',
+  callActivityFinished: 'call_activity_finished',
+  intermediateEventReached: 'intermediate_event_reached',
+  intermediateEventFinished: 'intermediate_event_finished',
+  boundaryEventReached: 'boundary_event_reached',
```

Comment:
Having looked at the implementation, I think it would make more sense, to only have a single `boundaryEventTriggerd` message, which gets send when the BoundaryEvent is executed.
BoundaryEvents don't really do anything, except redirecting the ProcessInstance's path down a new execution flow.

So I'd change this to: 

```suggestion
  boundaryEventTriggered: 'boundary_event_triggered',
```